.travis.yml

```diff
@@ -8,6 +8,7 @@ addons:
   postgresql: "9.3"
 before_install:
   - redis-server --version
+  - conda list
```

Comment:
@sjanssen2 apparently this repository is not set up with conda, can you either set it up to use conda (check the .travis.yml from the ag-web repo) or fix what is exactly the problem?